### PR TITLE
Build action runs on PR and doesn't publish bundle anymore

### PR DESCRIPTION
**Pull request by fhennig**
https://github.com/stackabletech/documentation-ui/issues/14

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -23,12 +21,4 @@ jobs:
             ${{ runner.OS }}-node-
             ${{ runner.OS }}-
       - run: npm ci
-      - run: node_modules/.bin/gulp lint
       - run: node_modules/.bin/gulp bundle
```
Comment (nightkr) [inline .github/workflows/build.yml:24]: I'd swap this around. `gulp lint` is still useful, `gulp bundle` not so much.

Comment (fhennig) [reply .github/workflows/build.yml:24]: bundle includes the lint step, and I thought why not run the whole bundling step? it just takes 3 seconds

Comment (fhennig) [reply .github/workflows/build.yml:24]: But I'd be fine with just running the linting too

Comment (nightkr) [reply .github/workflows/build.yml:24]: Ah yeah I see now. Fair enough.

**Review by nightkr — APPROVED** [review]

**Pull request merged**